flake.nix

```diff
@@ -174,14 +173,12 @@
           constituents = [
             self.hydraJobs.tarball
             self.hydraJobs.build.x86_64-linux
-            self.hydraJobs.build.i686-linux
```

Comment:
should we not rather exclude pre-commit for 32-bit? I think patchelf in nixpkgs ist still needed for 32-bit libs.